main.js

```diff
@@ -157,10 +157,10 @@ function isVisible(window, bounds) {
   const topClearOfUpperBound = window.y >= boundsY;
   const topClearOfLowerBound = (window.y <= (boundsY + boundsHeight) - BOUNDS_BUFFER);
 
-  return rightSideClearOfLeftBound
-    && leftSideClearOfRightBound
-    && topClearOfUpperBound
-    && topClearOfLowerBound;
+  return rightSideClearOfLeftBound &&
```

Comment:
I could go either way on these. But generally I think that the operator at the beginning of the line signals what's going on a little bit quicker. Maybe we should make that 'operator on next line' rule more global?

Reply:
Switching it to `before` caused more changes and then I even learned that our overrides (`"after", { "overrides": { "?": "before", ":": "before" } }`) is already the default so I changed it to just `'operator-linebreak': 'error',` 👍